registration.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const form = document.querySelector('form');
+const inputFullname = document.querySelector('input#fullname');
+const fullNameHelper = document.querySelector('div#fullname');
+const inputPhoneNumber = document.querySelector('input#phone');
+const phoneNumberHelper = document.querySelector('div#phone');
+const inputHiddenInfo = document.querySelector('input#hidden-info');
+const extraInfoHelper = document.querySelector('div#extra-info');
+const inputRegion = document.querySelector('select#region');
+const selectCities = document.querySelector('select#city');
+const cityHelper = document.querySelector('div#city');
+const submit = document.querySelector('input#submit');
```

Comment:
I means `const cityHelper = document.querySelector('#city');`
Also, if you use forms[0] instead of query selector(`const inputFullname = form.querySelector('input#fullname');`), you can use `form.fullname`